demos/array-shell-sort/README.md

```diff
@@ -0,0 +1,41 @@
+# Merge Sort Demo
+
+Description and guide for a randomized shell sort on element-wise summation of input lists with secure MPC.
+
+## Protocol
+Each party inputs an array of length n. The protocol sums these arrays element-wise, and outputs that summed list, sorted
+via a randomized shell sort. See shellsortprotocol.pdf for a full description of the algorithm and its implementation.
+
+## Running Demo
+
+1. Running a server:
+    ```shell
+    node demos/array-shell-sort/server.js
+    ```
+
+2. Either open browser based parties by going to *http://localhost:8080/demos/array-merge-sort/client.html* in the browser, or a node.js party by running 
```

Comment:
^ also here

Reply:
Local Software Engineer Doesn't Know How To Use Find-Replace. 
Resolved with last push.